### DISCUSSION TITLE
Add Resources Patch #311 Restoring

### DISCUSSION THREAD

**Discussion by anil-al** [ReVanced/revanced-discussions#557] [Question] [answered]
Tablets are showing excessively much adverts. 24 days ago, to fix this, it has been said ad resources patch to be restored which before called not necessary and deleted. Yet, for no reason, it hasn't been included again and it is waiting for 24 days. There's no news since then. Please do include this patch again.

**Comment by oSumAtrIX** [comment, marked as answer]
Downgrade the patches and tools and check if they fix the issue.

**Reply by anil-al** [reply]
I am not able to do it since I don't have the knowledge to do that but it has been said in the discussion regarding tablet ads that ads have been started/increased after deletion of that patch. That was why it was wanted to be restored but since then, there is no news.

**Reply by oSumAtrIX** [reply]
Further testing has to be made to make sure the addition does what we expect it to do.

**Reply by anil-al** [reply]
> Further testing has to be made to make sure the addition does what we expect it to do.

By the way, I found the comment regarding the effect of the patch: https://github.com/revanced/revanced-patches/issues/206#issuecomment-1193257100
